crmd/attrd.c

```diff
@@ -90,10 +90,12 @@ update_attrd_helper(const char *host, const char *name, const char *value,
     int attrd_opts = attrd_opt_none;
 
     if (is_remote_node) {
-#if HAVE_ATOMIC_ATTRD
         attrd_opts |= attrd_opt_remote;
-#else
-        /* Talk directly to cib for remote nodes if it's legacy attrd */
+
+#if !HAVE_ATOMIC_ATTRD
+        /* Legacy attrd can handle remote peer remove ('C') requests,
```

Comment:
crmd always talks to the local attrd, so not a problem. the (newer) local attrd will broadcast the request to all peers, but older ones will just ignore it, same as before. also, nothing essential needs to be done for peer remove in legacy attrd; i implemented it here mainly b/c it was so trivial, and it's a step toward supporting remote nodes.